fe/fe-core/src/main/java/org/apache/doris/qe/AuditLogHelper.java

```diff
@@ -122,12 +130,19 @@ public static String handleStmt(String origStmt, StatementBase parsedStmt) {
             }
         }
         if (rowCnt > 0) {
-            return origStmt + " ... /* total " + rowCnt + " rows, truncated audit_plugin_max_sql_length="
-                + GlobalVariable.auditPluginMaxSqlLength + " */";
+            // This is an insert statement.
+            int maxLen = GlobalVariable.auditPluginMaxInsertStmtLength <= 0
```

Comment:
making `maxlen = max(0, min(insert_sql_len, max_sql_len));` seems more straightforward.
no need to consider 0 or negative value for auditPluginMaxInsertStmtLength (default INT.MAX)